src/utils/validate.js

```diff
@@ -73,32 +73,72 @@ export const validateNewpassword = (newPasswordData, setErrors) => {
   return isValid;
 };
 
-export const validateAddTask = (addTaskData,setErrors) => {
-
-
+export const validateTask = (taskDetails, setFormErrors,type) => {
+  const { task_title, task_description, due_date } = taskDetails;
   let errors = {};
-  let isVaild = true;
-  if (!addTaskData.task_title.trim()) {
-    errors.task_title = "Task Name is Required";
-    isVaild = false;
-  }
-  if (!addTaskData.task_description.trim()) {
-    errors.task_description = "Task Description is Required";
-    isVaild = false;
+  let isValid = true;
+
+  if (!task_title.trim()) {
+    errors = { ...errors, task_title: `${type} name is required` };
+    isValid = false;
   }
- 
-  if (!addTaskData.due_date) {
-    errors.due_date = "Task Due Date is Required";
-    isVaild = false;
+
+  if (!task_description.trim()) {
+    errors = { ...errors, task_description: `${type} description is required` };
```

Comment:
Description with D in capital letter

NO need to show type before error message, as we're already in that page

Reply:
Changed it